plugins/php.json

```diff
@@ -10,10 +10,12 @@
   "env": {
     "PHPFPM_ERROR_LOG_FILE": "{{ .Virtenv }}/php-fpm.log",
     "PHPFPM_PID_FILE": "{{ .Virtenv }}/php-fpm.log",
-    "PHPFPM_PORT": "8082"
+    "PHPFPM_PORT": "8082",
+    "PHPRC": "{{ .DevboxDir }}"
```

Comment:
Does this break if file/directory are missing? (user might delete `devbox.d/php` and we don't recreate.

Reply:
Yes, if no `php.ini` is detected, php will use the default values for all directives.